packages/client/package.json

```diff
@@ -97,7 +97,7 @@
     "typescript": "5.2.2"
   },
   "engines": {
-    "node": "^18.15.0"
+    "node": ">=18 <21"
```

Comment:
Can you please add another changeset with this so it surfaces in the changelog?

Reply:
done